src/store/modules/auth.store.js

```diff
@@ -10,6 +11,36 @@ export const authStore = {
     namespaced: true,
     state: initialState,
     actions: {
+        setting({ dispatch, commit }, {
```

Comment:
please move this to actions of `store/modules/user.store.js`

Reply:
done